### PR TITLE
Remove filetime dep from build_helper

### PR DESCRIPTION
**Pull request by Mark-Simulacrum**
r? @alexcrichton

### CODE REVIEW

File: src/bootstrap/util.rs [hunk 1]
```diff
@@ -45,7 +45,6 @@ pub fn copy(src: &Path, dst: &Path) {
                dst.display(), e)
     }
     let metadata = t!(src.metadata());
-    t!(fs::set_permissions(dst, metadata.permissions()));
```
Comment (retep998) [inline src/bootstrap/util.rs]: This change seems unrelated?

Comment (Mark-Simulacrum) [reply src/bootstrap/util.rs]: Ah, right. It's not needed, but I forgot to remove it from this patch.
